app/assets/javascripts/temperature.js

```diff
@@ -1,26 +1,25 @@
 $(function () {
   'use strict';
-
   window.Temperature = window.Temperature || {};
 
   Temperature.setData = function (widget, data) {
-    widget.find('.widget-hotness').removeClass('hotness0');
-    widget.find('.widget-hotness').removeClass('hotness1');
-    widget.find('.widget-hotness').removeClass('hotness2');
-    widget.find('.widget-hotness').removeClass('hotness3');
-    widget.find('.widget-hotness').removeClass('hotness4');
+    widget.find('.widget-temperature').removeClass('temperature0');
+    widget.find('.widget-temperature').removeClass('temperature1');
+    widget.find('.widget-temperature').removeClass('temperature2');
+    widget.find('.widget-temperature').removeClass('temperature3');
+    widget.find('.widget-temperature').removeClass('temperature4');
 
-    var min = widget.find('.widget-hotness')[0].getAttribute('data-min');
-    var max = widget.find('.widget-hotness')[0].getAttribute('data-max');
+    var min = widget.find('.widget-temperature')[0].getAttribute('data-min');
+    var max = widget.find('.widget-temperature')[0].getAttribute('data-max');
```

Comment:
Probably best to store `widget.find('.widget-temperature')` into a variable, as you use it many times (and dom lookups are quite expensive).